bin/start-pgbouncer

```diff
@@ -64,6 +64,10 @@ run-pgbouncer() {
 }
 
 config-gen() {
+  if [ -f "vendor/pgbouncer/pgbouncer.ini" ]
```

Comment:
This change does mean that someone debugging config eg on a one-off dyno won't see the config change if they adjust an env var and re-run this script. However, this may be rare enough that it doesn't matter. I'll leave this up to you :-)

Reply:
It does, but I do think it is pretty rare. I couldn't think of a good way (admittedly I only spent a few minutes thinking about it) to provide a testable run without guarding against a re-write of the config from the wrapper.